integration-tests/src/jvmTest/kotlin/ai/koog/integration/tests/utils/MediaTestUtils.kt

```diff
@@ -618,42 +620,32 @@ object MediaTestUtils {
             """.trimIndent()
         }
 
-        val file = File(testResourcesDir, "test_${scenario.name.lowercase()}.md")
-        file.writeText(markdownContent)
+        val file = testResourcesDir.resolve("test_${scenario.name.lowercase()}.md")
+        Files.writeString(file, markdownContent)
         return file
     }
 
-    fun createAudioFileForScenario(scenario: MediaTestScenarios.AudioTestScenario, testResourcesDir: File): File {
+    fun createAudioFileForScenario(scenario: MediaTestScenarios.AudioTestScenario, testResourcesDir: Path): Path {
         return when (scenario) {
             MediaTestScenarios.AudioTestScenario.BASIC_WAV -> {
-                val file = File(testResourcesDir, "test.wav")
-                check(file.exists()) { "WAV test file should exist" }
+                val file = testResourcesDir.resolve("test.wav")
+                check(Files.exists(file)) { "WAV test file should exist" }
                 file
             }
 
             MediaTestScenarios.AudioTestScenario.BASIC_MP3 -> {
-                val file = File(testResourcesDir, "test.mp3")
-                if (!file.exists()) {
-                    val sourceFile = File(testResourcesDir, "test.mp3")
-                    check(sourceFile.exists()) { "MP3 test file should exist" }
-                    file.writeBytes(sourceFile.readBytes())
-                }
-                file
-            }
-
-            MediaTestScenarios.AudioTestScenario.BIG_AUDIO -> {
-                val file = File(testResourcesDir, "test_big.wav")
-                if (!file.exists()) {
-                    val sourceFile = File(testResourcesDir, "test_big.wav")
-                    check(sourceFile.exists()) { "Source WAV test file should exist" }
-                    file.writeBytes(sourceFile.readBytes())
+                val file = testResourcesDir.resolve("test.mp3")
+                if (!Files.exists(file)) {
+                    val sourceFile = testResourcesDir.resolve("test.mp3")
+                    check(Files.exists(sourceFile)) { "MP3 test file should exist" }
```

Comment:
are `sourceFile` and `file` the same? then how is it possible that `file` does not exist and `sourceFile` exists?